### PR TITLE
Created DemoSeederCommand to replace seeders

### LINKED ISSUE #884: Create a command for seeding demo data
So that we can provide an easier installation for newcomers, we need to move the seeding code into a command.

### PR DESCRIPTION
**Pull request by jbrooksuk**
Closes #884.

### CODE REVIEW

**Comment by jbrooksuk** [comment]
Once merged I'll need to update the cron job on our server.

**Comment by GrahamCampbell** [comment]
Our server should probably be updated to pull from 1.2, not master.

File: app/Console/Commands/DemoSeederCommand.php [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Seeds Cachet with demo data.';
+
+    /**
+     * Execute the console command.
```
Comment (GrahamCampbell) [inline app/Console/Commands/DemoSeederCommand.php]: missing phpdoc

**Comment by GrahamCampbell** [comment]
Don't add a .gitkeep file, just leave the DatabaseSeeder class there.

**Comment by GrahamCampbell** [comment]
Like in https://github.com/StyleCI/StyleCI/blob/master/database/seeds/DatabaseSeeder.php.

**Comment by jbrooksuk** [comment]
Done :)

**Pull request merged**